lib/pmp/pmp.go

```diff
@@ -116,6 +116,16 @@ func (w *wrapper) AddPortMapping(ctx context.Context, protocol nat.Protocol, int
 	return port, err
 }
 
+func (w *wrapper) TryAddPinhole(_ context.Context, _ nat.Protocol, _ int, _ string, _ time.Duration) (int, error) {
+	// NAT-PMP doesn't support pinholes.
+	return 0, errors.New("Adding IPv6 pinholes is unsupported on NAT-PMP")
```

Comment:
```suggestion
	return 0, errors.New("adding IPv6 pinholes is unsupported on NAT-PMP")
```

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
You're right